enrichments/trace/internal/elastic/span.go

```diff
@@ -434,6 +462,43 @@ func (s *spanEventEnrichmentContext) enrich(
 	if cfg.ProcessorEvent.Enabled && s.exception {
 		se.Attributes().PutStr(AttributeProcessorEvent, "error")
 	}
+	if s.exceptionType == "" && s.exceptionMessage == "" {
+		// Span event does not represent an exception
+		se.Attributes().PutStr(AttributeEventKind, "event")
```

Comment:
I think this should belong in the ES exporter. In the [mapping spec](https://github.com/elastic/opentelemetry-dev/blob/main/docs/design-decisions/ingest/mapping.md#span-events), we're saying that the span event name should be mapped to `attributes.event.name`. This shouldn't depend on whether or not enrichment is used.

Reply:
CC: @carsonip

Reply:
ack

Reply:
attributes.event.name is already handled in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter/model.go#L645-L647

Reply:
I actually meant to comment on the line below, sorry for the confusion. The `event.kind` is ECS specific (https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html), so this shouldn't be handled in the exporter. Ideally wouldn't add this at all if possible. Is the UI relying on this?

Reply:
> so this shouldn't be handled in the exporter

Ah, makes sense.

> Ideally wouldn't add this at all if possible. Is the UI relying on this?

I am not sure, when creating this PR I did a crude search on Kibana repo and found some usage of `event.kind` but I have also [asked on slack](https://elastic.slack.com/archives/C07K6S1GDJB/p1727456600867439) if it is used. Since this is supposed to be in ECS if required, I think we can unblock the PR. I will remove this if the UI team confirm that it is not used.

Reply:
Confirmed on slack that `event.kind` is not required so I have removed it from the PR.